vendor/ocamlformat_support/format_.mli

```diff
@@ -359,8 +359,8 @@ val pp_print_or_newline : formatter -> int -> int -> string -> string -> unit
   not just been split. Otherwise, print the second string. *)
 
 val pp_print_fits_or_breaks :
-  formatter -> string -> int -> int -> string -> unit
-(** [pp_print_fits_or_breaks fmt fits nspaces offset breaks] prints [fits] if
+  formatter -> int -> string -> int -> int -> string -> unit
```

Comment:
Shouldn't it be optional ? For compatibility.

Reply:
Done, but it could still change before being merged into upstream ocaml anyway.